ddtrace/span.py

```diff
@@ -165,17 +165,9 @@ def finish(self, finish_time=None):
             self.duration_ns = ft - (self.start_ns or ft)
 
         if self._context:
-            try:
-                self._context.close_span(self)
-            except Exception:
-                log.exception("error recording finished trace")
-            else:
-                # if a tracer is available to process the current context
-                if self.tracer:
-                    try:
-                        self.tracer.record(self._context)
```

Comment:
do we need to do anything with `Tracer.record`?

Reply:
Deprecate it? lol

Reply:
Done in 9100551